# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mathieu/xcdroast/20_error_write_tracks.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ru xcdroast-0.98alpha14-/src/io.c xcdroast-0.98alpha14/src/io.c
--- xcdroast-0.98alpha14-/src/io.c	2003-06-15 22:44:04.000000000 +0200
+++ xcdroast-0.98alpha14/src/io.c	2003-06-15 22:46:10.000000000 +0200
@@ -6614,7 +6614,7 @@
 				read_output_ctrl = 4;
 			} 
 			/* check Medium error/buffer underrun */
-			if (strncmp(tmp2,"Input/output error",18)) {
+			if (strncmp(tmp2,"Input/output error",18) == 0) {
 				read_output_ctrl = 5;
 			}
 		}
